ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/objectstore/StoreManager.java

```diff
@@ -145,20 +145,20 @@ private static final ObjectStoreAPI initStore(String name)
     {
         ObjectStoreEnvironmentBean storeEnvBean = BeanPopulator.getNamedInstance(ObjectStoreEnvironmentBean.class, name);
         String storeType = storeEnvBean.getObjectStoreType();
-        ObjectStoreAPI store;
 
         try
         {
-            store = ClassloadingUtility.loadAndInstantiateClass(ObjectStoreAPI.class, storeType, name);
+            ObjectStoreAPI store = ClassloadingUtility.loadAndInstantiateClass(ObjectStoreAPI.class, storeType, name);
+
+            store.start();
```

Comment:
Thank you for review! Yes, I still get NPE but it is caught earlier and user gets better info about what happened - "ARJUNA012135: Could not create Store type: com.arjuna.ats.internal.arjuna.objectstore.jdbc.JDBCStore". You can look at log files before and after patch which I have uploaded to the JBTM-2835.